### DISCUSSION TITLE
Manual DNS-SD device announcement is picked up by the Apple Home app, but no UDP communication attempts are made

### DISCUSSION THREAD

**Discussion by georgzepf** [matter-js/matter.js#695] [Q&A]
Hi all!

Running the provided example device on a Raspberry Pi and commissioning it using the Apple Home app works without any issues. 

I captured / looked at the DNS-SD announcement and now want to publish the exact same service manually. My UDP socket listening on all IPv6 and IPv4 addresses / port 5540 should then be able to capture any connection attempts made by the Home app to the advertised addresses, at least to my understanding.

This works as described when using a Google Nest Mini and the Google Home app on Android.

With an Apple HomePod Mini and the Home app on iOS, the service is discovered just fine but when monitoring the relevant UDP traffic on my iPhone, no attempts are made at reaching the addresses provided.

I tried to announce the service manually using the `dns-sd` utility on macOS, `avahi-publish` on Debian as well as using some Python logic together with the `python-zeroconf` package but without any luck.

I know this is a rather unusual need, but because the example device manages to do commissioning with the Home app just fine, I think this may still be a good place to ask about this.

Would appreciate it a lot if someone could just point me in the right direction. What is so different about the Home app on iOS in this situation?

Best

**Comment by Apollon77** [comment]
I think a disciussion is a better place for this then an issue.

So when I get you right then you tell that matter.js works just fine, but you now want to do these MDNS announcements yourself yb command line utils? Then please show in detail what you do and whats the output is. YOu could also try to run wireshark to verify what exactly they send
